### DISCUSSION TITLE
How do we declare global browser in typescript when using devtools protocol

### DISCUSSION THREAD

**Discussion by mighty98** [webdriverio/webdriverio#10865] [Technical Discussion]
I am working on a project that needs chrome devtools.
I have an initializer class where i want to create a remote instance
`global.I = await remote({ automationProtocol: 'devtools', capabilities: { browserName: 'chrome' } });`

But the problem is with type definition. 
Above code returns WebdriverIO.Browser type

In global.d.ts i have below line
`declare var I: import('webdriverio').Browser;`

But with this setup it throws below error
`Type 'Browser' is missing the following properties from type 'Browser': sessionStatus, sessionNew, sessionEnd, sessionSubscribe, and 25 more.ts(2740)`

Can someone please help

**Comment by christian-bromann** [comment]
> declare var I: import('webdriverio').Browser;

Please make sure to add `@wdio/globals/types` to your `tsconfig.json` types list which should enable you to use the `WebdriverIO.Browser` namespace. See also https://webdriver.io/docs/typescript
